src/hive_utils/madara_utils.rs

```diff
@@ -0,0 +1,152 @@
+use crate::starknet_client::constants::STARKNET_NATIVE_TOKEN;
```

Comment:
Should we take some time to replace all this by the ef-tests functions?

Reply:
probably yes in an upcoming cleaning PR